examples/next-graphql/components/helpers/ComponentResolver.tsx

```diff
@@ -1,12 +1,13 @@
 import { FragmentOf, readFragment } from "gql.tada";
 
-import ParagraphHero from "@/components/paragraph/ParagraphHero";
-import ParagraphCardGroup from "@/components/paragraph/ParagraphCardGroup";
-import ParagraphText from "@/components/paragraph/ParagraphText";
-import ParagraphImage from "@/components/paragraph/ParagraphImage";
-import ParagraphCodeBlock from "@/components/paragraph/ParagraphCodeBlock";
-import ParagraphStaticComponent from "@/components/paragraph/ParagraphStaticComponent";
-import ParagraphViewReference from "@/components/paragraph/ParagraphViewReference";
+// Drupal
+import ParagraphHero from "@/components/drupal/paragraph/ParagraphHero";
```

Comment:
maybe we can barrel this to have it like the next export

Reply:
agree on this, not sure if we can have a new PR to introduce DX improvements